### PR TITLE
Remove dependency on ellipsis

### PR DESCRIPTION
**Pull request by olivroy**
Since ellipsis imports rlang anyway. 

I also took the opportunity to remove duplicated package description. https://broom.tidymodels.org/reference/broom.html

### CODE REVIEW

**Comment by simonpcouch** [comment]
Thank you for the PR! Looking strong.

I'm seeing:

```
❯ checking dependencies in R code ... WARNING
  '::' or ':::' import not declared from: ‘ellipsis’
```

in our tests. Could you update this PR to address that warning and then I'll come back to review then?

**Comment by olivroy** [comment]
Appears that I forgot one. Sorry!

About the tests failures, they might be because Matrix and or lme4 are not installed correctly.  https://github.com/lme4/lme4/issues/699#issuecomment-1838926718

File: DESCRIPTION [hunk 1]
```diff
@@ -536,15 +535,14 @@ Description: Summarizes key information about statistical
     observations to a dataset, such as fitted values or influence
     measures.
 License: MIT + file LICENSE
-URL: https://broom.tidymodels.org/,
+URL: https://broom.tidymodels.org,
```
Comment (simonpcouch) [inline DESCRIPTION:538]:
```suggestion
URL: https://broom.tidymodels.org/,
```

File: DESCRIPTION [hunk 2]
```diff
@@ -1,4 +1,3 @@
-Type: Package
```
Comment (simonpcouch) [inline DESCRIPTION:1]: Let's restore this line as well.

**Review by simonpcouch — COMMENTED** [review]
Looking good! Will ask you to revert some unrelated changes and open separate issues if those changes are causing issues for you.

**Review by simonpcouch — APPROVED** [review]
Thank you!

**Pull request merged**